plugins/destination.go

```diff
@@ -88,6 +88,7 @@ func (p *DestinationPlugin) Migrate(ctx context.Context, tables schema.Tables) e
 	return p.client.Migrate(ctx, tables)
 }
 
+// nolint: unparam
```

Comment:
Not a blocker but we've discussed this internally and realized the linter was complaining since the private function `readAll` always got the same `sourceName` so technically we could remove that param and inline the value.

If we do that, we should probably move `readAll` to the testing file